cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -1,486 +0,0 @@
-          assertStats(r, systemUpdates = 1)
-        })) {
-          statsOnlyResultTable()
-          note {
-            p("[enterprise-edition]#The `SET STATUS {ACTIVE | SUSPENDED}` and `SET HOME DATABASE` parts of the command are only available in Enterprise Edition.#")
-          }
-        }
-        p("The created user will appear on the list provided by `SHOW USERS`.")
-        query("SHOW USERS YIELD user, suspended, passwordChangeRequired, roles, home WHERE user = 'jake'", assertUsersShown(Seq("jake"))) {
-          p(
-            """In this example we also:
-              |
-              |* Reorder the columns using a `YIELD` clause
-              |* Filter the results using a `WHERE` clause to show only the new user
-              |
-              |""".stripMargin
-          )
-          resultTable()
-        }
-        query("SHOW USERS YIELD roles, user WHERE \"PUBLIC\" IN roles RETURN user as publicUsers", ResultAssertions({ r: DocsExecutionResult =>
-          r.columnAs[String]("publicUsers").toSet should be(Set("neo4j", "jake"))
-          })) {
-          p(
-            """It is also possible to add a `RETURN` clause to further manipulate the results after filtering. In this
-              |case it is used to filter out the roles column and rename the users column to `publicUsers`.""".stripMargin)
-          resultTable()
-        }
-        note {
-          p(
-            """In Neo4j Community Edition there are no roles, but all users have implied administrator privileges.
-              |In Neo4j Enterprise Edition all users are automatically assigned the <<administration-security-roles-public, `PUBLIC`>> role, giving them a base set of privileges.""".stripMargin)
-        }
-        p("The `CREATE USER` command is optionally idempotent, with the default behavior to throw an exception if the user already exists. " +
-          "Appending `IF NOT EXISTS` to the command will ensure that no exception is thrown and nothing happens should the user already exist. " +
-          "Adding `OR REPLACE` to the command will result in any existing user being deleted and a new one created.")
-        query("CREATE USER jake IF NOT EXISTS SET PASSWORD 'xyz'", ResultAssertions(r => {
-          assertStats(r, systemUpdates = 0)
-        })) {
-          statsOnlyResultTable()
-        }
-        query("CREATE OR REPLACE USER jake SET PLAINTEXT PASSWORD 'xyz'", ResultAssertions(r => {
-          assertStats(r, systemUpdates = 2)
-        })) {
-          statsOnlyResultTable()
-          p("This is equivalent to running `DROP USER jake IF EXISTS` followed by `CREATE USER jake SET PASSWORD 'xyz'`.")
-        }
-        note {
-          p("The `IF NOT EXISTS` and `OR REPLACE` parts of this command cannot be used together.")
-        }
-        note {
-          p("User names are case sensitive.")
-        }
```

Comment:
This information is missing on the page https://neo4j.com/docs/cypher-manual/4.3/access-control/manage-users/#access-control-create-users.

Lets create a new PR that adds this information later on.